src/index.html

```diff
@@ -18,5 +31,82 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header>
+      <a href="logo" class="images">
+        <img
+        src="images/logo.png" alt="Moyo"
+        class="images"
+      />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a
+              href="Apple"
+              class="nav__link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li>
+            <a
+              href="Samsung"
+              class="nav__link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li>
+            <a
+              href="Smartphones"
+              class="nav__link"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li>
+            <a
+              href="Laptops&computers"
+              class="nav__link"
+              data-qa="hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li>
+            <a
+              href="Gadgets"
+              class="nav__link"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li>
+            <a
+              href="Tablets"
+              class="nav__link"
+            >
+              TABLETS
```

Comment:
Text transformation should be handled using CSS. Consider using CSS to transform text to uppercase instead of directly using uppercase letters in HTML.